src/components/customers/AddCustomerDrawer.tsx

```diff
@@ -1,10 +1,19 @@
-import { forwardRef, useEffect, useState, RefObject } from 'react'
-import styled from 'styled-components'
+import React, { forwardRef, useEffect, useState, RefObject } from 'react'
+import styled, { css } from 'styled-components'
 import { useFormik } from 'formik'
-import { object, string } from 'yup'
+import { array, object, string } from 'yup'
+import _, { FieldWithPossiblyUndefined } from 'lodash'
```

Comment:
Never import `_`
It prevents the tree shaking and will import the whole lodash lib while you only need one or 2 functions from it

Reply:
arf I thought it would do it anyway 😓 Thanks, will update

Reply:
Fixed